Iceberg-Plugin-GitHub.package/IceGitHubCreatePullRequestModel.class/instance/initializeWidgets.st

```diff
@@ -0,0 +1,26 @@
+initialization
+initializeWidgets
+
+	titleLabel := self newLabel.
+	titlePanel := self newTextInput autoAccept: true.
+	headLabel := self newLabel.
+	headForkList := self newDropList.
+	headBranchList := self newDropList.
+	baseLabel := self newLabel.
+	baseForkList := self newDropList.
+	baseBranchList := self newDropList.
+	bodyLabel := self newLabel.
+	bodyPanel := self newText autoAccept: true.
+		
+	titleLabel label: 'Title'.
```

Comment:
Static labels do not need a LabelPresenter anymore. You can just add a string in the layout.

Reply:
Why you can put strings in a layout? so the string is polymorphic with a presenter?

Reply:
If it is a String and not a Symbol representing the name of a variable containing a presenter, it will automatically consider it as a LabelPresenter.

Reply:
I think what pablo means is not "how does it work", but the fact that strings and presenters are polymorphic is confusing and potentially dangerous.

Reply:
What is currently done is that if the argument is neither a symbol or a layout, it will call #asPresenter on it. 

On strings, it will return a LabelPresenter. 

For now, most people who learned you can do that in Spec 2 seemed not confused and pretty happy with that because it simplifies a lot of presenters.

Reply:
I understand how it works, but thanks :).

I do not agree that it simplifies a lot (a couple of lines?).
And I do not agree that Iceberg needs that style of coding.

Reply:
Well, I added it :)
And I think Is a big simplification for coding form-style layouts, which are a lot of the dialogs you normally do. 
I do not understand why you need to do the double work of defining a presenter and assigning a label when you can do it directly in the layout avoiding the overhead.